parts/kubernetesmastercustomscript.sh

```diff
@@ -165,6 +167,11 @@ function writeKubeConfig() {
     chmod 700 $KUBECONFIGDIR
     chmod 600 $KUBECONFIGFILE
 
+    FQDNSuffix="cloudapp.azure.com"
+    if [ "$TARGET_ENVIRONMENT" = "AzureChinaCloud" ]
```

Comment:
Is this the only place the FQDN Suffix is used/needed? Otherwise we might want to compute it higher up rather than having this if/else down inside this script.